includes/class-sensei-utils.php

```diff
@@ -2513,6 +2513,39 @@ public static function show_course_lessons( $course_id ) {
 		 */
 		return apply_filters( 'sensei_course_show_lessons', true, $course_id );
 	}
+
+	/**
+	 * Check if a user is the lesson author.
+	 *
+	 * @since 3.0.2
+	 *
+	 * @param int $lesson_id
+	 * @param int $user_id
+	 * @return boolean
+	 */
+	public static function sensei_is_lesson_author( $lesson_id = 0, $user_id = 0 ) {
+
+		if ( ! $lesson_id || ! $user_id ) {
+			return false;
+		}
+
+		if ( (int) get_post_field( 'post_author', (int) $lesson_id ) === (int) $user_id ) {
```

Comment:
I don't think we need to cast `$lesson_id` to an `int`. We've already specified that we expect it to be an `int` [here](https://github.com/Automattic/sensei/pull/3080/files#diff-93297062f2e4dafc7f2edd50739e7219R2522).

Reply:
Done!